scripts/dockerfiles/Dockerfile.release

```diff
@@ -6,7 +6,9 @@ RUN yum update -y && \
 WORKDIR /app
 
 COPY aws-cni /app 
-COPY misc/aws.conf /app
+COPY misc/10-aws.conflist /app
+
+COPY portmap /app
```

Comment:
Can you remove this too?

Reply:
@micahhausler  Sure, I removed my last commit. It only handled portmap plugin. Let me know if you need further changes.